README.rst

```diff
@@ -18,17 +18,20 @@ Examples
 
    import json
 
+   from datetime import datetime, timedelta
+
    from jwt import (
        JWT,
        jwk_from_dict,
        jwk_from_pem,
    )
+   from jwt.utils import get_time
 
    message = {
        'iss': 'https://example.com/',
        'sub': 'yosida95',
-       'iat': 1485969205,
-       'exp': 1485972805,
+       'iat': get_time(datetime.utcnow()),
+       'exp': get_time(datetime.utcnow() + timedelta(hours=1)),
```

Comment:
I think it is a good idea to represent `iat` and `exp` in `datetime` for README readability, but this use of `get_time` is problematic (I will describe in a comment on `get_time`). I think it is better to use `datetime#timestamp` like `int(datetime.utcnow().timestamp())`.